tools/ports/__init__.py

```diff
@@ -275,7 +275,7 @@ def build_port(src_dir, output_path, port_name, includes=[], flags=[], cxxflags=
   @staticmethod
   def get_dir(*parts):
     dirname = os.path.join(config.PORTS, *parts)
-    shared.safe_ensure_dirs(dirname)
+    utils.safe_ensure_dirs(dirname)
```

Comment:
Looks like `safe_ensure_dirs` is changed as well, not just `path_from_root` and `WINDOWS`?

Reply:
Updated the title